pkg/postgres/walker/schema.go

```diff
@@ -157,8 +158,28 @@ func (s *Schema) SetOptionsMap(optionsMap search.OptionsMap) {
 
 // SetSearchScope sets search scope for the schema.
 func (s *Schema) SetSearchScope(searchCategories ...v1.SearchCategory) {
+	// TODO(ROX-28123): Clean up
+	normalizedSkipMap := set.NewStringSet(
```

Comment:
nit: Could these be consts?

Reply:
moved them to var section.  Apparently `set.NewStringSet` cannot be a const.